contracts/truefi2/TrueRateAdjuster.sol

```diff
@@ -27,13 +27,15 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
     using SafeMath for uint256;
     using TrueFiFixed64x64 for int128;
 
-    /// @dev basis precision: 10000 = 100%
     /// @dev maximum interest rate in basis points
     uint256 constant MAX_RATE_CAP = 50000;
```

Comment:
```suggestion
    uint256 constant MAX_RATE_CAP = 5 * BASIS_POINTS;
```
What do you think? You'll have to reorder lines

Reply:
Seems legit 👍